packages/edit-site/src/components/save-panel/index.js

```diff
@@ -151,9 +151,7 @@ export default function SavePanel() {
 			>
 				<Button
 					variant="secondary"
-					className={ clsx(
-						'edit-site-editor__toggle-save-panel-button'
-					) }
+					className={ 'edit-site-editor__toggle-save-panel-button' }
```

Comment:
Nit: double quotes are preferred when it's a string literal. 

```suggestion
					className="edit-site-editor__toggle-save-panel-button"
```

Reply:
Agreed. I believe there's an autofixable eslint rule for this we can enable. I'll send a separate PR.

Reply:
PR: https://github.com/WordPress/gutenberg/pull/62026